pinot-common/src/main/java/org/apache/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -203,6 +205,27 @@ public void setNumSegmentsMatched(long numSegmentsMatched) {
     _numSegmentsMatched = numSegmentsMatched;
   }
 
+  @JsonProperty("numConsumingQueried")
+  public long getNumConsumingQueried() {
+    return _numConsumingQueried;
+  }
+
+  @JsonProperty("numConsumingQueried")
+  public void setNumConsumingQueried(long queried) {
+    _numConsumingQueried = queried;
+  }
+
+  @JsonProperty("minConsumingIndexTs")
```

Comment:
Instead of 'Ts' could we suffix it with Millis (or Seconds, whatever the units may be)